### DISCUSSION TITLE
Fixed size element

### DISCUSSION THREAD

**Discussion by nzmedet** [SVG-Edit/svgedit#787] [New feature request]
Is there a way to disable shape element resizing?

**Comment by brettz9** [comment]
One can programmatically add selections without the resize grips (e.g., 2nd argument boolean for `selectOnly` and `addToSelection` of `svgEditor.canvas`), but I can see no way to disable resizing.

I'll label this issue as an "enhancement" as I presume that is what you are looking for.
